generic_config_updater/gu_common.py

```diff
@@ -750,6 +750,13 @@ def _get_path_tokens_from_leaf(self, model, token_index, xpath_tokens, config):
             # leaf_list_name = match.group(1)
             leaf_list_value = match.group(1)
             list_config = config[leaf_list_name]
+            # Workaround for those fields who is defined as leaf-list in YANG model but have string value in config DB
+            # No need to lookup the item index in ConfigDb since the list is represented as a string, return path to string immediately
+            # Example:
+            #   xpath: /sonic-buffer-port-egress-profile-list:sonic-buffer-port-egress-profile-list/BUFFER_PORT_EGRESS_PROFILE_LIST/BUFFER_PORT_EGRESS_PROFILE_LIST_LIST[port='Ethernet9']/profile_list[.='egress_lossy_profile']
+            #   path: /BUFFER_PORT_EGRESS_PROFILE_LIST/Ethernet9/profile_list
+            if isinstance(list_config, str):
+                return [leaf_list_name]
             list_idx = list_config.index(leaf_list_value)
```

Comment:
>list_config [](http://example.com/codeflow?start=23&length=11)

Do you want to add `else if` block? What is the expected type?

Reply:
The return type is a list of tokens (strings). I don't like adding else to an if that has `return`. I prefer to avoid indentation when possible.